plugins/TodayAtMun/__init__.py

```diff
@@ -0,0 +1,53 @@
+import discord
+from discord.ext import commands
+
+from Bot import bot
+from Plugin import AutomataPlugin
+from plugins.TodayAtMun.diary_parser import diary_parser
+from plugins.TodayAtMun.Today import Today
+
+
+class TodayAtMun(AutomataPlugin):
+    def __init__(self, manifest, bot):
+        super().__init__(manifest, bot)
+        self.Tod = Today()
+
+    @commands.group()
+    async def today(self, ctx):
+        """ Provides Brief Info Of Significant Dates on the Mun Calendar"""
+        if not ctx.invoked_subcommand:
+            await ctx.send("Please Provide Subcommand e.g !today [soon]")
+
+    @today.command(name="next")
+    async def today_next(self, ctx):
+        # Sends quick update on Muns Next Calendar Date on the Uni Diary
```

Comment:
Comments such as these are the start of functions are better suited as doc. strings instead, such as you are doing on the above `today` method

Reply:
I had this done. But went back on it for some reason. Will fix!